doc/user/content/sql/prepare.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "PREPARE"
+description: "`PREPARE` creates a prepared statement (a server-side object that can be used to optimize performance)."
```

Comment:
The note in parens is probably only correct for postgres, but not for us. It should be removed.